### ISSUE TITLE
Cleanup documentation for number_of_shards / number_of_replicas

### ISSUE THREAD

**Issue by dliappis** [elastic/rally-eventdata-track#69]
While reviewing #67 a few clean ups in the docs were identified namely:

https://github.com/elastic/rally-eventdata-track/pull/67#discussion_r368424368

https://github.com/elastic/rally-eventdata-track/pull/67#discussion_r368424865

https://github.com/elastic/rally-eventdata-track/pull/67#discussion_r368425026

https://github.com/elastic/rally-eventdata-track/pull/67#discussion_r368425526

https://github.com/elastic/rally-eventdata-track/pull/67#discussion_r368425761

https://github.com/elastic/rally-eventdata-track/pull/67#discussion_r368426461

In short, the defaults for `number_of_replicas` and `number_of_shards` get set in the index template and we should document the params and the default value as common for all tracks.
